PSFzf.psm1

```diff
@@ -1,3 +1,7 @@
+param(
+	[parameter(Position=0,Mandatory=$false)][string]$PSReadlineChordProvider = 'Ctrl+T',
+	[parameter(Position=1,Mandatory=$false)][string]$PSReadlineChordReverseHistory = 'Ctrl+R')
```

Comment:
I would probably avoid **silently** changing `Ctrl+R` behavior. In `fzf` case, user needs to opt-in for `bash` / `zsh` / `fish` `Ctrl+R` replacements.

People may not appreciate such change.

I would personally switch to fzf behavior, but it's better to be opt-in than opt-out.